internal/kldcontracts/rest2eth.go

```diff
@@ -419,12 +420,21 @@ func (r *rest2eth) resolveParams(res http.ResponseWriter, req *http.Request, par
 	c.msgParams = make([]interface{}, len(c.abiMethod.Inputs))
 	queryParams := req.Form
 	for i, abiParam := range c.abiMethod.Inputs {
-		if bv, exists := c.body[abiParam.Name]; exists {
+		argName := abiParam.Name
+		// If the ABI input has one or more un-named parameters, look for default names that are passed in.
+		// Unnamed Input params should be named: input, input1, input2...
```

Comment:
Going to dig around for precedent here in the code - I think we have a naming convention on start from 0 or 1 etc. that it's worth being consistent with...

Reply:
Aha, this is what we already have in `genOutput` 👍

Reply:
(bet it was me who set that precedent... feels a bit odd in retrospect, but ah well 🤷‍♂️ )